opentelemetry-jaeger/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## vNext
 
+## v0.21.0
+
+### Changed
+- Move opentelemetry-jaeger::Propagator logic to its own crate (opentelemetry-jaeger-propagator) to prepare for opentelemetry-jaeger exporter deprecation [#1487](https://github.com/open-telemetry/opentelemetry-rust/pull/1487)
```

Comment:
lets be more descriptive here to smoothen migration.
This crate previously contained two functionalities - an exporter to Jaeger, and a Jaeger propagator. Going forward, Jaeger propagator is shipped as new crate <crate-name>. The JaegerExporter is deprecated, and users should use OTLP exporter instead. See <linktoexample> showing OTLPExporter with Jaeger.

Reply:
> The JaegerExporter will be deprecated after the next release, and users should use OTLP exporter instead. See "linktoexample" showing OTLPExporter with Jaeger.

+1. Let's just call out this create will be deprecate with only exporter in it.